From aa1a55b272fda2873c7a3c9da56e8e89883cb0a6 Mon Sep 17 00:00:00 2001
From: Adam Seaton <aseaton@google.com>
Date: Thu, 18 Aug 2016 14:11:04 -0700
Subject: [PATCH 14/63] Update Security String to 2016-10-05 to Platform and
 CTS for October Security Updates (rebased)

b/30946906

Change-Id: Ia5207f919908496a593e3163788bfb75ce4e055d
(cherry picked from commit bb45bd20e97be07ec92facf411aae91d90490a5b)
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index ae12a06..31decf1 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -113,7 +113,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
   # Must be of the form "YYYY-MM-DD" on production devices.
   #
   # If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-  PLATFORM_SECURITY_PATCH := 2016-10-01
+  PLATFORM_SECURITY_PATCH := 2016-10-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.9.3

